core/storage/email/gae_models.py

```diff
@@ -442,4 +437,4 @@ def thread_id(self):
         Returns:
             str. The thread id.
         """
-        return self.id.split('.')[2]
+        return self.id[self.id.find('.') + 1:]
```

Comment:
This should not be here. It needs to be generalized.

All "exploration thread"-specific logic should go in one place in the codebase, and this sort of stuff should delegate to that instead. Otherwise there will be confusion because what purports to be a general-purpose method/class seems to be doing stuff that's only specific to one type of thread.

Reply:
I think this location is fine (this extracts the thread_id from the FeedbackEmailReplyToIdModel id). I will edit the above @property exploration_id to use the domain class function.

Reply:
Edited the above exploration_id property.